### DISCUSSION TITLE
Wake asymmetry FAST.Farm curl

### DISCUSSION THREAD

**Discussion by MYMahfouz** [OpenFAST/openfast#2169] [Q&A] [answered]
Dear all,

I have been using the curl model to simulate floating wind farm in FAST.Farm.

I have used the parameters as recommended in your most recent [publication](https://wes.copernicus.org/preprints/wes-2024-40/wes-2024-40.pdf). However, I am running the 15MW on a semisubmersible floater and not the 5MW.

I used the vtk files to extract the velocities at the hub height plane (XY plane at hub height), and drew the velocity deficits as shown below.

The Figure shows three turbines in a row behind each other with a distance of 6D between each turbine. The Figure represents the deficit at hub height behind each turbine at a distance of 5D. Turbine 2 sees the undisturbed inflow then turbine 5 is behind it and turbine 8 is at the end of the row.

The Figure shows an asymmetry in the wake deficit which i am unable to explain. Can you help me understand why the asymmetry exist? 

Thanks in advance

![image](https://github.com/OpenFAST/openfast/assets/33868271/f3ab7112-d53b-4091-85a2-48136dbac5f8)

Best regards,
Youssef

**Comment by jjonkman** [comment]
Dear @MYMahfouz,

The curled wake model of FAST.Farm naturally has inherent asymmetry, dependent on the inflow-skew angle.  You haven't stated what conditions you are simulating, but presumably the inflow relative to the rotor is skewed a bit (due to tilt, yaw, or floater motion).

Best regards,

**Reply by MYMahfouz** [reply]
Dear @jjonkman,

Thank you for your reply and for your time.

I am simulating normal operation conditions. The mooring system is a stiff design so the floater motions are minimum. The inflow turbulence intensity is 6%. I have checked different wind fields at different wind speeds and the asymmetry is visible for all cases I checked. The windfield is created using the turbulent mann box generator.

I assume the platform yaw has the biggest effect but it is small. The time series of the platform yaw is shown in the Figure below, where the mean yaw angle is around 1 degree. The range of the yaw amplitude is between 4 and 5 degrees. The roll angle is less than 1 degree. 

I checked now also the inflow skew angle `RtSkew` from AeroDyn outputs and it increases for downwind turbines. I also added a figure for it below. I do not know how this is defined in AeroDyn. Can you please explain to me how the `RtSkew` output channel is calculated? Is it the skew relative to the norm of the rotor area?

Is this enough to cause this asymmetry?

![image](https://github.com/OpenFAST/openfast/assets/33868271/6f32db7d-d8d8-4e63-af9d-41e2aacc0c14)


![image](https://github.com/OpenFAST/openfast/assets/33868271/fd61d16c-c4cb-409c-bc63-ac89073c57fd)


Best regards,

**Reply by jjonkman** [reply, marked as answer]
Thanks for clarifying.  Indeed, that is sizeable amount of skew, which will lead to an asymmetric wake.

The `RtSkew` output from AeroDyn is the inflow angle, i.e., the angle between the rotor-disk averaged relative wind velocity (including the effect of wind variation and structural velocity along each blade) and the axis normal to the rotor.  This skew is computed universally, and so, can include effects from both tilt and yaw.  While your yaw may be small, the tilt could be sizeable due to shaft tilt, tower deflection, and floater pitch, in combination with the yaw.

Best regards,
